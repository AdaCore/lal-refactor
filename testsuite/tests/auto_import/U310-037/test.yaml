description: This test checks that subprograms declared on child packages are handled correctly.

driver: shell_script
