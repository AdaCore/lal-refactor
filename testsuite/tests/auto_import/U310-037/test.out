Available Imports:
Import: Foo.Bar; Qualifier: Foo.Bar

Edits:
main.adb
2:1-2:1 with Foo.Bar;

5:4-5:4 Foo.Bar.

